ros2_socketcan_msgs/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>ros2_socketcan_msgs</name>
+  <version>1.1.0</version>
```

Comment:
Is this version intended or just mistakenly copied from `ros2_socketcan`?

Reply:
This is a copy from the current `ros2_socketcan`. We can reset this to 0.0.0 until it is released if you want.